theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
I thought it would not be necessary, since `GpuDimShuffle` inherits from `HideC`.
@abergeron did I miss something?
I don't really like the idea of an almost-empty file.